packages/destination-actions/src/destinations/google-analytics-4/selectPromotion/index.ts

```diff
@@ -22,11 +22,31 @@ const action: ActionDefinition<Settings, Payload> = {
         }
       }
     },
+    creative_name: {
+      label: 'Creative Name',
+      type: 'string',
+      description: 'The name of the promotional creative.'
+    },
+    creative_slot: {
+      label: 'Creative Slot',
+      type: 'string',
+      description: 'The name of the promotional creative slot associated with the event.'
+    },
     location_id: {
       label: 'Location ID',
       type: 'string',
       description: 'The ID of the location.'
     },
+    promotion_id: {
+      label: 'Promotion ID',
+      type: 'string',
+      description: '	The ID of the promotion associated with the event.'
```

Comment:
Lets remove the initial tab/spacing
```suggestion
      description: 'The ID of the promotion associated with the event.'
```